### PR TITLE
Add testnet build flag

### PR DESCRIPTION
**Pull request by bemusementpark**
Pass in `-testnet` or `-Ptestnet` in Android Studio to set the app to use `testnet`.

testnet is `http://public.loki.foundation:38157`

Otherwise we use the usual seed nodes, usually these:

```
https://seed1.getsession.org:4443
https://seed2.getsession.org:4443
https://seed3.getsession.org:4443
```

### CODE REVIEW

File: libsession/build.gradle [hunk 1]
```diff
@@ -9,6 +9,9 @@ android {
 
     defaultConfig {
         minSdkVersion androidMinimumSdkVersion
+
+        def useTestnet = project.hasProperty('testnet')
```
Comment (AL-Session) [inline libsession/build.gradle:13]: Can we have some usage instructions here?

```suggestion
        // To enable testnet usage, from Android Studio select File > Settings > Build, Execution, Deployment > Gradle-Android Compiler
        // and enter a Command-line Option of `-testnet` and then build the application.
        // With testnet enabled the seed node will be http://public.loki.foundation:38157 and any accounts created or messages sent
        // will not be persisted on the network, while on the real network the seed nodes are are follows:
        //  - https://seed1.getsession.org:4443
        //  - https://seed2.getsession.org:4443
        //  - https://seed3.getsession.org:4443
```

**Review by AL-Session — APPROVED** [review]
Love it! Great work! =D

**Review by simophin — APPROVED** [review]
LGTM

**Comment by ThomasSession** [comment]
This has been moved to a debug menu where the environment ca be switched dynamically.

**Pull request closed without merging**